src/sagemaker/djl_inference/model.py

```diff
@@ -716,6 +872,8 @@ def __init__(
         self.enable_cuda_graph = enable_cuda_graph
         self.triangular_masking = triangular_masking
         self.return_tuple = return_tuple
+        self.save_mp_checkpoint_path = None
+        self.checkpoint = None
```

Comment:
If user has a partitioned model that they didn't partition with sagemaker, they may want to specify this, right? I don't think we need to figure this out in this PR, but right now I don't think we support that use-case (maybe i'm wrong).

Reply:
No you are right. In that case, we need to provide a way using PySDK. They could still work it out with their own serving.properties for now.

Reply:
Resolved